src/main/resources/com/parasoft/findings/sonar/res/messages.properties

```diff
@@ -20,3 +20,6 @@ RuleDocNotFoundAt=Rule documentation not available at {0}
 RuleNotFound=Rule {0} is not active or does not exist in {1}. Set a Quality profile that includes Parasoft rules as active.
 UnknownRuleName=Unknown rule
 UnsupportedFindingLanguage=Ignoring finding: {0} for unsupported language: {1}
+CoverageReportPathName=Parasoft coverage report path
```

Comment:
Please change this to "Parasoft Coverage Report Files" to be consistent with style of other titles:
1. "files" instead of "path"
2. Upper case for each word